10_temperature_object/temperature_object_spec.rb

```diff
@@ -24,44 +24,44 @@
   describe "can be constructed with an options hash" do
     describe "in degrees fahrenheit" do
       it "at 50 degrees" do
-        Temperature.new(:f => 50).in_fahrenheit.should == 50
+        Temperature.new(:fahrenheit => 50).in_fahrenheit.should == 50
       end
 
       describe "and correctly convert to celsius" do
         it "at freezing" do
-          Temperature.new(:f => 32).in_celsius.should == 0
+          Temperature.new(:fahrenheit => 32).in_celsius.should == 0
```

Comment:
`:fahrenheit => 32` this syntax was common somewhere during the WW2, as I recall
`fahrenheit: 32` is a cure